src/streaming/models/CmcdModel.js

```diff
@@ -269,23 +269,47 @@ function CmcdModel() {
         return keys.filter(key => enabledCMCDKeys.includes(key));
     }
 
+    function _isIncludedInRequestFilter(type){
+        const {includeInRequests} = getCmcdParametersFromManifest();
+        const includeInRequestsArray = includeInRequests.split(' ');
+
+        if(!includeInRequests || includeInRequestsArray.find(include => include === '*')){
+            return true;
+        }
+ 
+        const filtersTypes = {
+            'segment': HTTPRequest.INIT_SEGMENT_TYPE | 
```

Comment:
Maybe we can just put here the SEGMENT types that are included in getCmcdData method. Those are INIT_SEGMENT_TYPE and MEDIA_SEGMENT_TYPE. Because the other one are not included in the If statements.

Reply:
I agree, done!